nominal/_config.py

```diff
@@ -45,12 +45,15 @@ def get_token(self, url: str) -> str:
         raise NominalConfigError(f"url {url!r} not found in config: set a token with `nom auth set-token`")
 
 
+NominalConfig = NominalConfigV1
```

Comment:
Is this used elsewhere, or in scripts, or why is `NominalConfig` retained and set to V1?

Reply:
Yeah I mean it was there before so it would be a breaking change to remove it, not possible to know what exists out in the wild. Though was this originally a private module? if so, I guess no reason to keep it